### PR TITLE
feat:added copy of cfgFile in table psql

### PR DESCRIPTION
**Pull request by DmitriyLewen**

### CODE REVIEW

File: dbservice/postgresdb/dbservice_test.go [hunk 1]
```diff
@@ -639,3 +639,113 @@ func TestInsertInTableName(t *testing.T) {
 		t.Errorf("Unexpected error in 'insertInTableName', expected: %v, got: %v", badUpdateError, err)
 	}
 }
+
+func TestInsertCfgCacheSource(t *testing.T) {
+	t.Log("happy insert")
+	savedPsqlConnect := psqlConnect
+	psqlConnect = func(connectUrl string) (*sqlx.DB, error) {
```
Comment (AndreyLevchenko) [inline dbservice/postgresdb/dbservice_test.go:646]: could you make it parametrized and reuse `psqlConnect` code - meaning part of test data can be a function for sql config

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: dbservice/postgresdb/dbservice_test.go (hunk 1 above)
Comment (DmitriyLewen) [reply dbservice/postgresdb/dbservice_test.go]: done

File: dbservice/postgresdb/dbservice_test.go [hunk 2]
```diff
@@ -110,532 +110,185 @@ func TestInitError(t *testing.T) {
...
-	t.Log("bad update")
-	badUpdateError := errors.New("bad update")
-	psqlConnect = func(connectUrl string) (*sqlx.DB, error) {
-		db, mock, err := sqlxmock.Newx()
-		if err != nil {
-			log.Println("failed to open sqlmock database:", err)
+func TestInsert(t *testing.T) {
+	tests := []struct {
+		name          string
+		wasQuaryError bool
```
Comment (AndreyLevchenko) [inline dbservice/postgresdb/dbservice_test.go:192]: `Quary` -> `Query` (in 2 properties)

File: dbservice/postgresdb/dbservice_test.go [hunk 3]
```diff
@@ -110,532 +110,185 @@ func TestInitError(t *testing.T) {
...
+					return db, err
+				}
+				defer func() {
+					psqlConnect = savedPsqlConnect
+				}()
+
+				psqlDb, _ := psqlConnect(db.ConnectUrl)
+				err := runInsertFunc(psqlDb, insertFunc)
+				if err != nil {
+					if test.expectedError == nil || err.Error() != test.expectedError.Error() {
```
Comment (AndreyLevchenko) [inline dbservice/postgresdb/dbservice_test.go:234]: ` !errors.Is(err, expectedError)`

**Review by AndreyLevchenko — CHANGES_REQUESTED** [review]

File: dbservice/postgresdb/dbservice_test.go (hunk 3 above)
Comment (DmitriyLewen) [reply dbservice/postgresdb/dbservice_test.go]: Done

File: dbservice/postgresdb/dbservice_test.go (hunk 2 above)
Comment (DmitriyLewen) [reply dbservice/postgresdb/dbservice_test.go]: Done

**Review by AndreyLevchenko — APPROVED** [review]

**Pull request merged**